src/index.html

```diff
@@ -7,12 +7,59 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="block"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="photo"
+      />
+      <p class="block__apple">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="block__product">Product code: 195434</p>
+      <div class="block__stars">
+        <div class="stars stars--4">
+          <div class="stars__stars"></div>
+          <div class="stars__stars"></div>
+          <div class="stars__stars"></div>
+          <div class="stars__stars"></div>
+          <div class="stars__stars"></div>
+        </div>
+        <p class="block__reviews">Reviews: 5</p>
+      </div>
+      <div class="block__price">
+        <p class="block__price-txt">Price:</p>
+        <p class="block__costs">$2,199</p>
+      </div>
+      <a
```

Comment:
Add the blank line between the blocks
```suggestion
      </div>
      
      <div class="block__price">
        <p class="block__price-txt">Price:</p>
        <p class="block__costs">$2,199</p>
      </div>
      
      <a
```